src/foam/core/PropertyInfo.java

```diff
@@ -18,6 +18,7 @@
   public String getName();
   public Object get(Object obj);
   public void set(Object obj, Object value);
+  public String of();
```

Comment:
Why does PropertyInfo need to have this method? Only some sub-interfaces/classes of ProeprtyInfo will need this, not all.